Automattic/jetpack
# Support Card: Change plan check to feature check

#### Changes proposed in this Pull Request:

* Support Card: Change plan check to feature check

#### Jetpack product discussion


#### Does this pull request change what data or activity we track or use?
* No

#### Testing instructions:
* Use any method you'd like to get the changes on a Jetpack site connected to WP.com. I used both Jetpack Docker + Building and sending the changes to a WoA dev site.
* Visit the Jetpack dashboard. ( `/wp-admin/admin.php?page=jetpack#/dashboard` )
* On sites without a plan, you should see: "Jetpack offers support via community forums ... "
* On sites with the support feature (Basically any plan personal or higher), you should see: "Your paid plan gives you access to prioritized Jetpack support."

![2022-04-29_11-57](https://user-images.githubusercontent.com/937354/165989659-a45eef83-5cb9-4595-b21d-e8489cfcbfdf.png)
![2022-04-29_11-57_1](https://user-images.githubusercontent.com/937354/165989670-0b6ac415-da0e-4a26-b5c2-866cf2a1f1db.png)


Should behave the same before and after this PR.
